### PR TITLE
Support Bitcoin

### PR DESCRIPTION
**Pull request by maxwells**
Added Bitcoin Receiver & Transaction objects

### CODE REVIEW

**Comment by maxwells** [comment]
WIP

File: lib/resources/BitcoinReceivers.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+var StripeResource = require('../StripeResource');
+var stripeMethod = StripeResource.method;
+
+module.exports = StripeResource.extend({
+
+  path: 'bitcoin/receivers',
+
+  includeBasic: [
+    'create', 'list', 'retrieve'
```
Comment (michelle) [inline lib/resources/BitcoinReceivers.js]: Can't update, correct?

File: lib/resources/BitcoinReceivers.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  ],
+
+  listBitcoinTransactions: stripeMethod({
+    method: 'GET',
+    path: '/{receiverId}/transactions',
```
Comment (michelle) [inline lib/resources/BitcoinReceivers.js]: Do BitcoinTransactions not need their own resource? Can't retrieve by ID/update?

File: test/resources/BitcoinReceivers.spec.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+    });
+
+  });
+
+  describe('listBitcoinTransactions', function() {
+
+    it('Sends the correct request', function() {
+
+      stripe.bitcoinReceivers.listBitcoinTransactions('receiverId', {
```
Comment (michelle) [inline test/resources/BitcoinReceivers.spec.js]: Is this naming redundant? `listBitcoinTransactions`

**Comment by michelle** [comment]
Few notes, mostly due to me not having state on the API, but LGTM!

File: lib/resources/BitcoinReceivers.js (hunk 1 above)
Comment (maxwells) [inline lib/resources/BitcoinReceivers.js]: correct. they're only create/list-able by API users.

File: lib/resources/BitcoinReceivers.js (hunk 2 above)
Comment (maxwells) [inline lib/resources/BitcoinReceivers.js]: The API is exposed, but after chatting with ray & christian about it, we decided there wasn't a strong story for adding it to the bindings as a top level object. They are useful as a sublist of a receiver, though so I stuck with that.

File: test/resources/BitcoinReceivers.spec.js (hunk 3 above)
Comment (maxwells) [inline test/resources/BitcoinReceivers.spec.js]: Yeah. `listTransactions` would be better. I'll touch that up.

Thanks for reviewing these changes!

File: lib/resources/BitcoinReceivers.js (hunk 2 above)
Comment (michelle) [inline lib/resources/BitcoinReceivers.js]: Is there any functionality we'd be missing out on w/r/t transactions?

Comment (maxwells) [inline lib/resources/BitcoinReceivers.js]: they're immutable from the user API, so not that I can think of. These transactions represent the transfer of bitcoins to fill the receiver before it's used in a charge -- they're very specific to the receiver itself.

Comment (michelle) [inline lib/resources/BitcoinReceivers.js]: :100:

**Pull request merged**